earth_orography/README.md

```diff
@@ -0,0 +1,34 @@
+# Earth orography (ETOPO2022 Dataset)
+
+
+## Overview 
+
+This artifact provides the 30 and 60 arc-second ice-surface orography information 
+in NetCDF format. 
+- ETOPO_2022_v1_30s_N90W180_surface.nc (30 arc-second, ice surface, 1.5GB)
+- ETOPO_2022_v1_60s_N90W180_surface.nc (60 arc-second, ice surface, 456MB)
```

Comment:
Thanks, it's very helpful to have the size in the readme. We should do it everywhere.